src/sentry/lang/javascript/processor.py

```diff
@@ -187,18 +187,32 @@ def fetch_release_file(filename, release):
     cache_key = 'releasefile:v1:%s:%s' % (
         release.id,
         md5(filename).hexdigest(),
-    )
+    ),
+
+    filename_path = None
+    try:
+        # Can we guarantee filename is always parsable?
```

Comment:
I think we can. I'm not aware of a case where it would raise. 

BUT, it's possible that `parsed_url.path` may be identical to `filename_path` in which case we should check that case and not attempt to fetch two identical idents.